lib/stub_ui/compositing.dart

```diff
@@ -215,7 +215,7 @@ class SceneBuilder {
   /// Android view: When resizing an Android view there is a short period during
   /// which the framework cannot tell if the newest texture frame has the
   /// previous or new size, to workaround this the framework "freezes" the
-  /// texture just before resizing the Android view and unfreezes it when it is
+  /// texture just before resizing the Android view and un-freezes it when it is
```

Comment:
I thought US English preferred unhyphenated variants (e.g. Commonwealth 'co-ordinate', vs US 'coordinate'). 

The unhyphenated version looks more natural to me and I suspect we'll see this re-introduced (see what I did there) if we don't make this sort of thing a presubmit.

Reply:
My understanding of US English hyphenation is that we use it when a term is less common, but we do use it when the unhyphenated version (see what I did there? FYI, Chrome flagged unhyphenated as a spelling error...) becomes mainstream, and a word unto itself.  In this case "unfreeze" didn't seem common enough to me, but I could totally be persuaded to not hyphenate: it was just a subjective judgement call.